extension/src/popup/views/AccountHistory/index.tsx

```diff
@@ -30,12 +30,16 @@ const getIsPayment = (type: Horizon.OperationResponseType) =>
     Horizon.OperationResponseType.pathPaymentStrictSend,
   ].includes(type);
 
+const getIsSwap = (operation: HistoryItemOperation) =>
+  operation.type_i === 13 && operation.source_account === operation.to;
+
+enum SELECTOR_OPTIONS {
```

Comment:
unrelated changed, I moved this enum outside the AccountHistory component. I noticed it was causing a re-render loop from the useEffect below (only noticed bc I added a log for debugging)